common/templates/deployment.yaml

```diff
@@ -16,7 +16,10 @@ spec:
     {{- toYaml .Values.strategy | nindent 6 }}
   template:
     metadata:
-      {{- include "common.podAnnotations" . | nindent 6 }}
+      {{- if .Values.podAnnotations }}
```

Comment:
This isn't what we want, because if we don't define Values.podAnnotations, the checksum/config annotation won't be applied